doc/src/manual/arrays.md

```diff
@@ -392,8 +392,10 @@ julia> x = collect(reshape(1:9, 3, 3))
  2  5  8
  3  6  9
 
-julia> x[1:2, 2:3] = -1
--1
+julia> x[1:2, 2:3] .= -1
```

Comment:
A few lines above we say we're going to show that `=` is lowered to `setindex!`: the text should somewhat be adapted to explain why one needs to use `.=` here. Maybe better show a single-element assignment first.